### DISCUSSION TITLE
Automatic shutdown if inactivity on selected channels

### DISCUSSION THREAD

**Discussion by amane93** [rtl-airband/RTLSDR-Airband#282] [Q&A]
Is it possible that in scan/multichannel mode if theres inactivity on all channels for a specific time rtl_airband automatically shuts down?

**Comment by szpajder** [comment]
Nope, no such feature is currently implemented. The program is meant for
continuous operation, regardless of the activity on monitored channels.

**Reply by amane93** [reply]
OKay thanks!
